### PR TITLE
Fixed CREATE LIKE multi-db

### PR DESCRIPTION
**Pull request by Hydrocharged**
Fixes https://github.com/liquidata-inc/dolt/issues/654

### CODE REVIEW

File: go/libraries/doltcore/sqle/sqlselect_test.go [hunk 1]
```diff
@@ -709,7 +709,7 @@ var BasicSelectTests = []SelectTest{
 				"0o2rnf5pq2s1nq3hj3609e1lt0socuf1",
 				"billy bob",
 				"bigbillieb@fake.horse",
-				time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC),
+				time.Date(1970, 1, 1, 0, 0, 0, 0, &time.Location{}),
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/sqlselect_test.go:712]: Hmm, this might be the sign of a bug.

**Review by zachmu — APPROVED** [review]
LG, but I'm suspicious I broke something by deleting convert_dates. Let me know if I should dig in.

File: go/libraries/doltcore/sqle/sqlselect_test.go (hunk 1 above)
Comment (Hydrocharged) [reply go/libraries/doltcore/sqle/sqlselect_test.go:712]: So I I did a test by referencing the commit immediately before the `triggers 3` PR merge, and then the PR's commit, and this issue only popped up after the PR was merged. So it's definitely from that.

**Comment by Hydrocharged** [comment]
It looks like some of the failing tests have two issues. 1) They're wrong. Naked variables only work with system variables, and they're being used here for user variables, 2) Changing them to user variables causes this line to be hit (https://github.com/liquidata-inc/go-mysql-server/blob/fff5edda928de40709eec9b7b4f0559464560dfe/sql/plan/set.go#L78) with the following error: `unsupported type for set: *expression.UserVar`

So I'll just leave this PR as-is until we get those fixed, since this PR is basically just updating the `go-mysql-server` version and adding a bats test of previously-broken behavior.

**Comment by Hydrocharged** [comment]
A separate PR fixed many of the issues previously in this PR, so this is just a bats test now.

**Pull request merged**